<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>impqtClass</class>
 <widget class="QMainWindow" name="impqtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>impqt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="inputEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>441</width>
      <height>701</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>439</width>
       <height>699</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="outputEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>361</width>
      <height>831</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
